.github/workflows/openjdk_14_build.yml

```diff
@@ -0,0 +1,117 @@
+name: openjdk_14_build
+on:
+  pull_request:
+  push:
+    branches: [ main, develop ]
+    tags: [ v* ]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: gcr.io/swirlds-registry/cci-openjdk-ghr:14.0.2
+    env:
+      MAVEN_OPTS: -Xmx5200m
+      LC_ALL: C.UTF-8
+      DEBIAN_FRONTEND: noninteractive
+    timeout-minutes: 60
+    steps:
+      - run: apt-get -y install software-properties-common
+      - run: add-apt-repository ppa:git-core/ppa
+      - run: apt-get -y install git
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: mvn --version
```

Comment:
This command looks like leftover debug output. You can always add this flag to one of the below mvn commands if it's useful, otherwise remove.